ocaml/testsuite/tests/backtrace/names.ml

```diff
@@ -106,7 +106,9 @@ let inline_object f =
 
 let[@inline never] lazy_ f =
   let x = Sys.opaque_identity (lazy (1 + f ())) in
-  Lazy.force x
+  Lazy.force x [@tail]
+  (* CR less-tco: Force this to be TCO'd so that it does not show up in the stacktrace.
```

Comment:
Usually when a CR describes an action, that action is something we want to do but haven't. I'd rephrase the background about the problem and how we fix it to be descriptive, and then add something about returning to the test to see if there's a nicer way to make the outputs consistent.

Reply:
Done.